public/js/components/ArticleButtons.js

```diff
@@ -14,7 +15,7 @@ export default class ArticleButtons extends React.Component {
     callVote(vote) {
         if (!this.canVote()) return 
         ArticleActions.postVote(vote, this.props.id)
-            .then(ArticleActions.fetchArticles)
+            .then(() => ArticleActions.fetchArticles(ArticleStore.getSort(), ArticleStore.getLastArticle()))
```

Comment:
I didn't like how I had to use the store in the component just to get the params for fetch articles, if you guys can think of a better flux to fix this that'd be great